src/Lykke.Service.LoggingAdapter/Helpers/ErrorResponceBuilder.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Lykke.Common.Api.Contract.Responses;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+namespace Lykke.Service.LoggingAdapter.Helpers
+{
+    public static class ErrorResponceBuilder
+    {
+        public static ErrorResponse CreateErrorResponce(this ModelStateDictionary modelState)
```

Comment:
https://github.com/LykkeCity/Lykke.Common.ApiLibrary/blob/master/src/Lykke.Common.ApiLibrary/Contract/ErrorResponseFactory.cs#L18

btw, there is a spelling error - respon**s**e